tests/Unit/Config/Drupal8Test.php

```diff
@@ -313,6 +313,7 @@ public function testConfigDefault(): void
                 'sort_algorithm' => 'alpha',
             ],
             'ordered_imports' => [
+                'case_sensitive' => TRUE,
```

Comment:
just fix case here as GrumPHP says in CI then would merge.

Reply:
```suggestion
                'case_sensitive' => true,
```

Reply:
Change done.